### PR TITLE
Spam updates

### PR DESCRIPTION
**Pull request by mekline**
A new page on scammer policies, and accompanying changes to other pages (e.g. getting started, compensation language) to incorporate how those policies will be implemented.

### CODE REVIEW

File: docs/source/community-study-approval-process.rst [hunk 1]
```diff
@@ -80,20 +120,21 @@ When your study is ready and you’ve responded to the peer feedback,
 you can :ref:`submit your study <study status>` for Lookit approval so it can go live.
 This is the point where you'll need to have your access agreement set up.
 
-When you submit the study, you will be prompted to note any non-standard elements that require specific approval per the Terms of Use (e.g., integration of
+When you submit the study, you will be prompted to note any non-standard elements 
+that require specific approval per the Terms of Use (e.g., integration of
 additional about participants from another source) as well as what
```
Comment (bleonar5) [inline docs/source/community-study-approval-process.rst:125]: should this be additional "information"?

Comment (mekline) [reply docs/source/community-study-approval-process.rst]: indeed! fixed

File: docs/source/researchers-spam-prevention.rst [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+from your IRBs perspective, while *also* having them do the video consent frame 
+on Lookit/CHS (thus, you are not changing what counts as consent for your study, 
+you are merely implementing a platform-required fraud-prevention step before
+they even get to your study and the consent your study uses).
+
+If you discover that none of the above
+options is workable for you, we encourage you to think creatively about how else you
+could meet the requirement to verify that your participants are who they say they 
+are (or consider Option 1, volunteer participants.) It may help to inform your 
+IRB that other methods of identitiy verification (such as requiring a picture of a
```
Comment (bleonar5) [inline docs/source/researchers-spam-prevention.rst]: *identity*

File: docs/source/researchers-spam-prevention.rst [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+
+Option 4: Propose something else
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We understand that the procedures we describe here may not work for every study, or may
+require you to request a modification from your IRB. We encorage you to think about how
+to frame this requirement to your IRB, because it may well be considered a small or 
+negligible change from their perspective. For example, you could continue to use 
+whatever consent process you have for your study as being the official consent 
+from your IRBs perspective, while *also* having them do the video consent frame
```
Comment (bleonar5) [inline docs/source/researchers-spam-prevention.rst]: IRB's

**Review by bleonar5 — APPROVED** [review]
added some typo fixes in a new commit. everything looks good, clear, maximally informative. the idea of white hat exercises is very interesting.

**Pull request merged**